packages/pwa-kit-dev/src/utils/script-utils.test.js

```diff
@@ -242,7 +315,7 @@ describe('scriptUtils', () => {
 
             // De-code and re-encode gives the same result, to show that it *is* b64 encoded
             expect(Buffer.from(bundle.data, 'base64').toString('base64')).toEqual(bundle.data)
-        })
+        }, 25000)
```

Comment:
increased the timeout of this test as it fails in Github checks. I believe this is because of the increased time from getting pwa-kit dependencies using `npm ls --all`

Reply:
Rather than increasing the timeout, maybe just mock `execSync` to return mock data.